Blog-Post/src/scripts/validation/signInValidation.js

```diff
@@ -0,0 +1,26 @@
+import ValidationError from './validationError.js';
+
+const signInValidation = (username, password) => {
```

Comment:
Note: You can create a Validator class and keep all the validation logic there, or you can have a folder named 'validators' with validation functions. Both approaches are fine, just different—OOP and functional